android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormBaseActivity.java

```diff
@@ -88,7 +88,12 @@ protected void onCreate(Bundle savedInstanceState) {
         }
     }
 
-    private String getForm() {
```

Comment:
This does not always give a translated form, it can return the original form if the `JsonFormConstants.PERFORM_FORM_TRANSLATION` is set to false.

So the renaming is a bit misleading.